gnomad_qc/v5/sample_qc/merge_project_meta.py

```diff
@@ -261,45 +253,50 @@ def update_ht_to_final_schema(ht: hl.Table) -> hl.Table:
     return ht.select(*final_fields)
 
 
-def get_sample_collisions(meta_hts: Dict[str, hl.Table]) -> Set[str]:
+def get_sample_collisions(meta_hts: Dict[str, hl.Table]) -> hl.Table:
     """
     Get sample ID collisions between projects.
 
     :param meta_hts: Project metadata tables.
-    :return: List of sample IDs that exist in more than one project.
+    :return: Table of sample IDs that exist in more than one project.
     """
     # Get all possible pairs of projects
     project_pairs = list(combinations(meta_hts.keys(), 2))
-    sample_collisions = hl.empty_set(hl.tstr)
+
+    sample_collisions = hl.Table.parallelize([], schema=hl.tstruct(s=hl.tstr), key="s")
 
     for p1, p2 in project_pairs:
         logger.info(f"project 1 is {p1}, project 2 is {p2}")
         collisions_ht = meta_hts[p1].filter(hl.is_defined(meta_hts[p2][meta_hts[p1].s]))
-        collisions = collisions_ht.aggregate(hl.agg.collect_as_set(collisions_ht.s))
-        sample_collisions = sample_collisions.union(collisions)
+        sample_collisions = sample_collisions.union(collisions_ht.select())
+
+    sample_collisions = sample_collisions.checkpoint(
```

Comment:
just for consistency with other code and since it's more of a maintained table rather than intermediate: could move the checkout to the main right after this function is called

Reply:
Yes great point!